data/dukeData.txt

```diff
@@ -0,0 +1,2 @@
+todo | false | sleep
```

Comment:
I don't think this file needs to be committed, since it is dependent on the user input and does not add anything to the actual working of the program.